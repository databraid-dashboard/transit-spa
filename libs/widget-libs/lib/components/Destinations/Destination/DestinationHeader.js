var _jsxFileName = '../../src/components/Destinations/Destination/DestinationHeader.jsx',
    _this = this;

import React from 'react';
import PropTypes from 'prop-types';

import { Header } from 'semantic-ui-react';
import '../destination.css';

var DestinationHeader = function DestinationHeader(_ref) {
  var name = _ref.name;
  return React.createElement(
    Header,
    { as: 'h1', className: 'destination-header', __source: {
        fileName: _jsxFileName,
        lineNumber: 8
      },
      __self: _this
    },
    name
  );
};

DestinationHeader.propTypes = {
  name: PropTypes.string.isRequired
};

DestinationHeader.defaultProps = {
  name: ''
};

export default DestinationHeader;