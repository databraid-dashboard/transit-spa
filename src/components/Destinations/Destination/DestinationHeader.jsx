import React from 'react';
import PropTypes from 'prop-types';

import { Header } from 'semantic-ui-react';

import '../destination.css';

const DestinationHeader = ({ name }) => (
  <Header as="h3" className="destination-header">
    {name}
  </Header>
);

DestinationHeader.propTypes = {
  name: PropTypes.string.isRequired,
};

DestinationHeader.defaultProps = {
  name: '',
};

export default DestinationHeader;
