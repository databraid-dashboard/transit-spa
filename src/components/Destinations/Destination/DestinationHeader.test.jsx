import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DestinationHeader from './DestinationHeader';

describe('DestinationHeader', () => {
  it('should render when passed required props', () => {
    const name = 'Tartine';
    const component = shallow( <DestinationHeader name={name} />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
