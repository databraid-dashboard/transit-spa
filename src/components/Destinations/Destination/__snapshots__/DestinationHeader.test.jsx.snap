// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DestinationHeader should render when passed required props 1`] = `
<Header
  as="h3"
  className="destination-header"
>
  Tartine
</Header>
`;
